### ISSUE TITLE
Tab Module disabling item not working when using tabular class

### ISSUE THREAD

**Issue by sptessier** [Semantic-Org/Semantic-UI#3578]
When using the "ui top attached **tabular** menu" classes the item **disabled** class doesn't work

```
<div class="ui top attached tabular menu">
    <a class="active item" data-tab="first">First</a>
    <a class="item disabled" data-tab="second">Second</a>
    <a class="item disabled" data-tab="third">Third</a>
</div>
<div class="ui bottom attached active tab segment" data-tab="first">
    First
</div>
<div class="ui bottom attached tab segment" data-tab="second">
    Second
</div>
<div class="ui bottom attached tab segment" data-tab="third">
    Third
</div>
```

If I change the div class to be "ui top attached menu" the disabled class is applied has expected on menu items.

**See also:**
https://jsfiddle.net/wqo0a69n/

**Comment by IonutBajescu** [comment]
Closed as duplicate of #3178.

**Comment by lubber-de** [comment]
Fixed in upcoming https://github.com/fomantic/Fomantic-UI 2.7.3
See https://jsfiddle.net/o6mpnjL1/2/
